components/autocomplete_selector.tsx

```diff
@@ -32,47 +42,41 @@ export default class AutocompleteSelector extends React.PureComponent {
         listComponent: SuggestionList,
         listPosition: 'top',
     };
+    suggestionRef?: React.RefObject<HTMLDivElement>;
 
-    constructor(props) {
+    constructor(props: Props) {
         super(props);
 
         this.state = {
             input: '',
         };
     }
 
-    onChange = (e) => {
+    onChange = (e: {target: {value: string}}) => {
```

Comment:
You can be more specific here by using `ChangeEvent` from React
```suggestion
    onChange = (e: ChangeEvent<HTMLInputElement>) => {
```

Reply:
Resolved